docs/build/tutorials/nodes-and-staking/set-up-node-with-installer.md

```diff
@@ -239,9 +254,58 @@ avalanche/1.1.1 [network=mainnet, database=v1.0.0, commit=f76f1fd5f99736cf468413
 Done!
 ```
 
+## Advanced node configuration
+
+Without any additional arguments, the script installs the node in a most common configuration. But the script also enables various advanced options to be configured, via the command line prompts. Following is a list of advanced options and their usage:
+
+* `admin` - [Admin API](../../avalanchego-apis/admin.md) will be enabled
+* `archival` - disables database pruning and preserves the complete transaction history
+* `db-dir` - use to provide the full path to the location where the database will be stored
+* `fuji` - node will connect to Fuji testnet instead of the mainnet
+* `index` - [Index API](../../avalanchego-apis/index-api.md) will be enabled
+* `ip` - use `dynamic`, `static` arguments, of enter a desired IP directly to be used as the public IP node will advertise to the network
+* `rpc` - use `any` or `local` argument to select any or local network interface to be used to listen for RPC calls
+* `version` - install a specific node version, instead of the latest. See [here](set-up-node-with-installer.md#using-a-previous-version) for usage.
```

Comment:
Can you add links to each of the config flags in https://docs.avax.network/build/references/avalanchego-config-flags?